src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -522,6 +522,20 @@ export class ResultsViewPageStore {
             },
             { fireImmediately: true }
         );
+
+        reaction(
```

Comment:
Not sure if ResultsViewPageStore is the right place to trigger the gene selection.

Reply:
Instead of doing this we can wait `genes` to complete before rendering the `Mutations` tab, and then set an initial value for the observable selectedGene inside `Mutations.tsx` constructor. We should avoid using reactions anyway unless it is absolutely necessary.